### DISCUSSION TITLE
Asking for permission to add ConstructionWand translation to original mod

### DISCUSSION THREAD

**Discussion by Theta-Dev** [CFPAOrg/Minecraft-Mod-Language-Package#2042] [Q&A]
Hello CFPA team!
I am the developer of the [ConstructionWand](https://github.com/Theta-Dev/ConstructionWand/issues) mod and would like to include your Chinese translation:

https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/projects/1.18/assets/construction-wand/constructionwand/lang/zh_cn.json

However you chose to publish your translations under a more restrictive license (CC BY-NC-SA), while ConstructionWand is published under the MIT license. So I cant use your work in my project without permission.

Would you allow me to include your translation in my mod? I will credit your project in the readme file and on the in-game mod info page.

**Comment by baka-gourd** [comment]
@mudapi @beiai @WuzgXY-GitHub 
In fact, multiple licenses can be used within a repository
For example, you can specify that a particular file or directory uses a special license

**Comment by n0099** [comment]
assign @TartaricAcid

**Comment by AnzhiZhang** [comment]
Related issue: https://github.com/Theta-Dev/ConstructionWand/issues/53

**Comment by AnzhiZhang** [comment]
Thanks for your asking!

Actually, there is a new PR to update the translation: https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/2025, it is better to authorize the translation to you after this merged, and you could have the best translation.

For the authors, if #2025 merged, there are about 6 origin authors. @beiai (56c67993c131e8e53429f8323f4de76e52af5298, ca40d6b0fd839119c3e98fee2346d255a1ad9857, a779cb3cf2063dd431c6a21a58b1fcaaf11cf3ce), @WuzgXY-GitHub (bd1c2568ee4b3239e4a5b657ce83053316e32668), @mudapi (467adbf36d7bcd2644add1042c936bfaabd23fbb), @XianYuAmiao (29105549acae5758e9fdd624f6559e414f5508e1), @nai200 (29105549acae5758e9fdd624f6559e414f5508e1), @AnzhiZhang (29105549acae5758e9fdd624f6559e414f5508e1), and @mamaruo (29105549acae5758e9fdd624f6559e414f5508e1)

There is an issue talking about CFPA license, right now it takes all origin authors to agree can license a translation to another license. #2053.

However, thank you for mentioning us to consider our liscense and legal issues!

Best Wishes,
Andy Zhang

**Reply by AnzhiZhang** [reply]
This PR has merged.

According to this issue: #2053, I think it takes all authors to agree. Here is my license, and I guess other authors would like to reply here for license.

---

I have contributed to the chinese translation in [CFPAOrg](https://github.com/CFPAOrg)'s project [Minecraft-Mod-Language-Package](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package) of mod [Construction Wand](https://github.com/Theta-Dev/ConstructionWand), which is the file [zh_cn.json](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/29105549acae5758e9fdd624f6559e414f5508e1/projects/1.16/assets/construction-wand/constructionwand/lang/zh_cn.json). I consent my contribution in this translation licensed to [@Theta-Dev](https://github.com/Theta-Dev) in his mod [Construction Wand](https://github.com/Theta-Dev/ConstructionWand) for a long-term duration. He could use my translations without [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license limits.
